frontend/test/metabase/scenarios/custom-column/reproductions/21513-cc-confusion-field-function.cy.spec.js

```diff
@@ -0,0 +1,33 @@
+import {
+  restore,
+  popover,
+  openProductsTable,
+  enterCustomColumnDetails,
+} from "__support__/e2e/cypress";
+
+describe.skip("issue 21513", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should handle cc with the same name as an aggregation function (metabase#21513)", () => {
+    openProductsTable({ mode: "notebook" });
+    cy.findByText("Summarize").click();
```

Comment:
Nit:
This works and is probably even more elegant for this case, but just FYI there is also a `summarize` helper.

`summarize({ mode: "notebook" });`